frappe/sessions.py

```diff
@@ -178,10 +178,13 @@ def get():
 	}
 
 	bootinfo["desk_theme"] = frappe.get_cached_value("User", frappe.session.user, "desk_theme") or "Light"
+	bootinfo["absolute_date_format"] =  frappe.get_cached_value("User", frappe.session.user, "show_absolute_datetime_foramt") or 0
```

Comment:
Check `frappe.boot.user`. Better to retrieve it from that instead of duplicating data here.

reference:

https://github.com/frappe/frappe/blob/74a31b31ba13dbbf9d6929444e17508f99e2cb31/frappe/utils/user.py#L213-L233